apps/site/src/lib/components/Navbar/Navbar.tsx

```diff
@@ -34,6 +35,9 @@ function Navbar({ identity }: NavbarProps) {
 		window.addEventListener("scroll", scrollHandler);
 	}, []);
 
+	const deadlineSeconds = 1705305540;
```

Comment:
Issue: Can we move this to a utility function (maybe in the `lib/` directory) to reduce the repetition across the two files?

Reply:
Or alternatively, I think we could export the utility function from the Apply page.